dallinger/experiment_server/experiment_server.py

```diff
@@ -75,6 +78,14 @@ def Experiment(args):
 # primary recruiter's route:
 app.register_blueprint(recruiters.mturk_routes)
 
+# Load dashboard routes and login setup
+app.register_blueprint(dashboard.dashboard)
```

Comment:
Would it be possible to make `dashboard` and `dashboard_tabs` callables? Executing code on import has caused a lot of pain in tests in the past.

Reply:
Answer is "no" for `dashboard`, since it's a Flask Blueprint decorator, but maybe the import-time Flask stuff can be usefully separated as the dashboard functionality grows.

Reply:
How do you envision the `dashboard_tabs` callable working? Maybe we can make it a singleton class of some sort? I guess it's sort of similar to the config object, but I'm not sure I understand where issues might come up with the current pattern.

Reply:
If the pain I was worried about was going to hit, I think it would have been in the tests for `experiment_server.py`, where the importing happens. Since that was fine, I was probably worried unnecessarily.